client/app/l10n/en.json

```diff
@@ -641,17 +641,17 @@
   "The file will be available at URL: /s/$filename": "The file will be available at URL: /s/$filename",
   "The following step by step wizard will guide you through configuring your whistleblowing site.": "The following step by step wizard will guide you through configuring your whistleblowing site.",
   "The following steps have errors:": "The following steps have errors:",
-  "The HTTPS certificate loaded on the platform is expiring or has already expired.": "The HTTPS certificate loaded on the platform is expiring or has already expired.",
+  "The HTTPS certificate loaded on the platform is expiring or has already expired.": "The HTTPS certificate loaded on the platform is, or has already expired.",
   "The key code is either invalid or the submission has expired.": "The key code is either invalid or the submission has expired.",
-  "The manual configuration wizard will guide you through the setup of HTTPS from an alternative Certificate Authority.": "The manual configuration wizard will guide you through the setup of HTTPS from an alternative Certificate Authority.",
+  "The manual configuration wizard will guide you through the setup of HTTPS from an alternative Certificate Authority.": "The wizard will guide you through manually configuring HTTPS and getting it validated by an alternative certificate authority.",
```

Comment:
This correction is not correct: We offer a manual and an automatic configuration; so this slang should be preserved in the string that describe the specific mode. thank you anyway @comradekingu for the correction and i will update the context on transifex.